popout.js

```diff
@@ -237,119 +323,180 @@ class PopoutModule {
         });

```

Comment:
This `window.addEventLister("unload"` could probably be done once and just close all the popped out windows, instead of having a separate event listener per popout.